### PR TITLE
SSPI support for Windows through request header

### PR DESCRIPTION
**Pull request by kalexand-rh**
From https://trello.com/c/vdwkL6C7/909-document-sccfsi-windows-oc-kerberos-support

@enj PTAL

### CODE REVIEW

File: install_config/configuring_authentication.adoc [hunk 1]
```diff
@@ -1053,6 +1053,42 @@ a value is used as the display name. Optional, case-insensitive.
...
+endif::[]
+
+Starting in version
+ifdef::openshift-enterprise[]
+3.11,
+endif::[]
+ifdef::openshift-origin[]
+1.11,
+endif::[]
+SSPI connection support for request header connections is enabled.
```
Comment (enj) [inline install_config/configuring_authentication.adoc]: Starting in version 3.11, `oc` supports the Security Support Provider Interface (SSPI) to allow for SSO flows on Windows. If you use the request header identity provider with a GSSAPI enabled proxy to connect an Active Directory server to OpenShift Container Platform, users can automatically authenticate to OpenShift Container Platform through the `oc` command line interface from a domain-joined Windows computer.

**Comment by enj** [comment]
Slightly different wording, otherwise LGTM.

**Comment by kalexand-rh** [comment]
Thanks @enj!

@stuartchuan, will you PTAL?

@openshift/team-documentation PTAL

File: install_config/configuring_authentication.adoc [hunk 2]
```diff
@@ -1053,6 +1053,43 @@ a value is used as the display name. Optional, case-insensitive.
...
 
+[discrete]
+[[windows-sspi-using-request-header]]
+==== SSPI connection support on Windows
+
+ifdef::openshift-enterprise[]
+
+[IMPORTANT]
+====
+Using SSPI connection support on Windows is a Technology Preview feature.
```
Comment (ahardin-rh) [inline install_config/configuring_authentication.adoc]: I would say Microsoft Windows to ensure proper branding, but otherwise looks good! :+1:

**Comment by kalexand-rh** [comment]
Thanks @ahardin-rh!

**Pull request merged**